lib/cosmos/packets/packet.rb

```diff
@@ -629,7 +629,7 @@ def reset
     #   buffer of data and processors
     def clone
       packet = super()
-      if packet.instance_variable_get("@processors")
+      if packet.instance_variable_get("@processors".freeze)
         packet.instance_variable_set("@processors", packet.processors.clone)
         packet.processors.each do |processor_name, processor|
```

Comment:
According to https://github.com/SamSaffron/memory_profiler/issues/13 it is optimized in Ruby 2.1 but less than 2.1 requires a constant. Should we just focus on 2.1 and make that the new Ruby in the COSMOS windows installer?